api/posters/statistics/send.ts

```diff
@@ -0,0 +1,33 @@
+import dayjs from 'dayjs'
+
+import {logger} from '@nebula/log'
+
+import {
+  getCreatedPostersBetween,
+  getEmailContent,
+  RawPosterData,
+  sendEmail,
+} from './helpers'
+
+export const sendPosterStatistics = async () => {
+  try {
+    const now = dayjs().toDate()
+    const oneWeekBefore = dayjs().subtract(1, 'week').toDate()
+
+    logger.info(
+      `Retrieving posters from ${oneWeekBefore.toUTCString()} until ${now.toUTCString()}`,
+    )
+
+    const posterDocuments: RawPosterData[] = await getCreatedPostersBetween(
+      oneWeekBefore,
+      now,
+    )
+
+    const totalPosters = posterDocuments.length
+    const emailContent = getEmailContent(posterDocuments, totalPosters)
+
+    await sendEmail(emailContent)
+  } catch (e) {
+    logger.error(`Error sending posters analytics`, e)
+  }
```

Comment:
We should reject the promise for observability. We can use something like `return Promise.reject(createHttpError(500, 'ERROR sending poster statistics'))`

Reply:
Great suggestion!